cmd/manager/main.go

```diff
@@ -182,3 +190,7 @@ func addMetrics(ctx context.Context) {
 		log.Error(err, "Failed to configure OSD metrics")
 	}
 }
+
+func healthCheck(req *http.Request) error {
+	return nil
```

Comment:
Is there a case where we want to check the response here? Does this check basically just enable ping?

Reply:
for now, it's just enabling ping check which is similar to [healthz.Ping](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.9.6/pkg/healthz/healthz.go#L186-L187) However, we can fill this with validations as a checker function

Reply:
Cool, makes sense